content/kubermatic/master/tutorials_howtos/CCM_migration/_index.en.md

```diff
@@ -26,11 +26,14 @@ Since the Kubernetes community has planned to deprecate and then remove all the
 controller managers and the Kubernetes documentation explain how to migrate from in-tree to out-of-tree CCM, KKP itself
 needed a mechanism to allow users to migrate their clusters to the out-of-tree implementation, as detailed below.
 
-The supported cloud providers so far are:
-* Hetzner
+The CCM/CSI migration is supported only for the following providers so far:
 * Openstack
 * vSphere
 
+> NOTE: latest vSphere CSI driver release (v2.3.0) does not support Kubernetes 1.22, hence it is not possible to:
```

Comment:
It's also not possible to create new 1.22 clusters on vSphere.

Reply:
Yes, but it is not related to ccm migration. That's why I didn't mention it here

Reply:
Yes, but someone might think if I can't migrate it, I'll just create a new cluster, which is not possible. That's why I thought about mentioning it.

Reply:
ok, added